---a/net/minecraft/block/BlockBarrier.java
+++b/net/minecraft/block/BlockBarrier.java
@@ -4,15 +4,12 @@
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BlockBarrier extends Block
-{
+
+public class BlockBarrier extends Block {
+
     private static final String __OBFID = "CL_00002139";
 
-    protected BlockBarrier()
-    {
+    protected BlockBarrier() {
         super(Material.barrier);
         this.setBlockUnbreakable();
         this.setResistance(6000001.0F);
@@ -20,20 +20,14 @@
         this.translucent = true;
     }
 
-    public int getRenderType()
-    {
+    public int getRenderType() {
         return -1;
     }
 
-    public boolean isOpaqueCube()
-    {
+    public boolean isOpaqueCube() {
         return false;
     }
 
-    @SideOnly(Side.CLIENT)
-    public float getAmbientOcclusionLightValue()
-    {
-        return 1.0F;
+    public void dropBlockAsItemWithChance(World worldIn, BlockPos pos, IBlockState state, float chance, int fortune) {
     }
-
-    public void dropBlockAsItemWithChance(World worldIn, BlockPos pos, IBlockState state, float chance, int fortune) {}
-}
+}
